up-l1/mqtt_5.adoc

```diff
@@ -24,165 +24,216 @@ MQTT is an OASIS standard messaging protocol for the Internet of Things (IoT). I
 
 For more information, please refer to https://mqtt.org/
 
-This document will discuss the uTransport implementation on MQTT 5.
+This document will discuss the uTransport implementation on MQTT 5 for two well known use cases:
+1. *Device-2-Device (D2D) Communication:* This is when devices (through the streamer) was to send messages to other devices through a cloud gateway (broker). 
```

Comment:
From my reading then a uStreamer would only need put the MQTT5 transport into a special mode and do nothing else special.

Is that correct?